articles/connections/database/custom-db/index.md

```diff
@@ -56,7 +56,7 @@ Here are some things to know before you begin the process of setting up your dat
 
 * [Update Users Using Your Database](/user-profile/customdb) has information on updating user profile fields.
 
-<%= include('../../../_includes/_ip_whitelist') %>
```

Comment:
I'm curious why we wouldn't want to include this? Is there a better place we could include this for custom databases?

Reply:
Nothing wrong with including the include there. The review app was erroring out, so I was just testing whether it was perhaps to do with the include.

Reply:
Heroku was apparently just being testy. The review app looks good now.